packages/ui/Icon/Icon.js

```diff
@@ -188,8 +186,8 @@ const Icon = kind({
 					} else if (iconProp.indexOf('0x') === 0) {
 						// Converts a hex reference in string form
 						icon = String.fromCodePoint(iconProp);
-					} else if (isSingleCharacter(iconProp)) {
-						// A single character is assumed to be an explicit icon string
+					} else if (!isUri(iconProp)) {
```

Comment:
Let's add some unit tests the valid `icon` values since we're updating this.

Reply:
Added several tests, which include testing this new approach.